docs/website/content/2.configuration/4.block/api.md

```diff
@@ -34,7 +34,7 @@ values: [
   {
     "name": "base_path",
     "type": "string",
-    "default": "\"\"",
+    "default": "",
```

Comment:
What was wrong with that? The default base path is empty, therefore `""`.

Reply:
This value gets also written to the search db and we can handle a visual fallback within a frontend component. e.g. render `default: ""` as `""`. Further type based improvements etc.
Clean data layer :)

Reply:
<img width="451" alt="Screenshot 2022-08-02 at 15 56 32" src="https://user-images.githubusercontent.com/1841067/182392607-892ae45b-f969-47be-a714-6a73f894dd6f.png">

fine for me, also object default is `-` instead of `{}`